project/OmnidocBuild.scala

```diff
@@ -22,12 +22,39 @@ object OmnidocBuild extends Build {
   val playSlickVersion         = sys.props.getOrElse("play-slick.version",         "4.0.0-RC2")
   val maybeTwirlVersion        = sys.props.get("twirl.version")
 
-  // these dependencies pull in all the others
+  // List Play artifacts so that they can be added as dependencies
```

Comment:
Not all projects are included, and because of that, some of the subprojects docs aren't being included. For example, there are no scaladocs for EhCache classes.